src/Tickets/Commerce/Hooks.php

```diff
@@ -211,6 +312,31 @@ public function pre_filter_admin_order_table( $query ) {
 		return $query;
 	}
 
+	/**
+	 * Validates a date range.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $date_from The start date.
+	 * @param string $date_to The end date.
+	 *
+	 * @return bool
+	 */
+	protected function is_valid_date_range( string $date_from = '', string $date_to = '' ): bool {
+		if ( empty( $date_from ) || empty( $date_to ) ) {
+			return true;
+		}
+
+		$date_from_ts = strtotime( $date_from );
+		$date_to_ts   = strtotime( $date_to );
+
+		if ( $date_to_ts >= $date_from_ts ) {
+			return true;
+		}
+
+		return false;
```

Comment:
At this point, we could simplify this to a single line:

```suggestion
		return $date_to_ts >= $date_from_ts;
```